docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,2182 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        return yyyymmdd.cast("date")
</pre><p>This entire function can be turned into a one-line:</p><p> </p><pre class="ql-syntax" spellcheck="false">return val.cast(str).lpad(8, "0").to_timestamp("%m%d%Y").date()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>uggghhhhh dang I looked for this, and I think I found to_timestamp() but then ignored it because it returned a TimeStamp, I didn't make the connection that then I merely had to call .date() to extract the Date.</p><p> </p><p>Probably this is just me, but perhaps this is a documentation issue? Or perhaps DateValue and TimeStampValue should .parse_string() classmethods? When I went to the docs I went to these datatypes at the start of my search but I didn't find anything there.</p>

Reply:
I think it's a documentation issue. We need a cookbook of sorts, or at least some categorical (e.g., Timestamp Operations) how-to content.